src/Snppts/Snippets/TodoMe.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Snppts.Snippets;
+
+public class TodoMe : IAmASnippet
+{
+    public string Slug => "TodoMe";
+
+    public string Title => "Todo.me";
+
+    public string Description => "Todo.me is a simple note taking and task tracking app made with .NET MAUI.";
+
+    public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("behl1anmol/Todo.me");
+
+    public Uri ExternalUri => null;
+
+    public bool ContainsAndroidSample => true;
+
+    public bool ContainsiOSSample => false;
+
+    public bool ContainsUWPSample => false;
+
+    public Sdk BuiltWith => Sdk.Maui;
+
+    public IAmAnAuthor AuthorInfo => new AnmolBehl();
+
+    public IEnumerable<Uri> ImageUris => new List<Uri>
+    {
+        new Uri("https://github.com/behl1anmol/Todo.me/blob/master/Screenshots/Maui_app_todo.gif"),
```

Comment:
Can you change this one to point directly to the image instead of the GitHub page?  Thanks!

Reply:
I have made the necessary changes. 
Regards,
Anmol